help/md/dnGamma.md

```diff
@@ -3,9 +3,11 @@ dnGamma
 ## title
 Gamma Distribution
 ## description
-Gamma probability distribution for positive real numbers.
+The Gamma probability distribution describes the probability of the sum of exponentially distributed variables. 
 ## details
-The gamma distribution is the probability of the sum of exponentially distributed variables. Thus, it provides a natural prior distribution for parameters that could be considered as sums of exponential variables.
+The gamma distribution takes on two parameters, shape and rate. Similar to how 1/rate gives the mean of the exponential, shape/rate gives the mean of the gamma. It provides a natural prior distribution for parameters that could be considered as sums of exponential variables.
+
```

Comment:
Could it be worth rephrasing this a little bit to make it clear that Gamma(shape = 1, rate) reduces to Exp(rate)?